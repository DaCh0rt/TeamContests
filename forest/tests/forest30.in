836 958
3726 1024
3544 -1879
-8871 -6137
2031 -6326
-7066 -7012
-1165 2511
6941 -7248
7007 -8489
9681 931
4827 -5039
-8997 4591
3264 -486
5955 6723
7781 5031
-9630 -6198
4780 934
-5162 -9706
-6882 6868
-6071 4776
-3847 -9756
886 4619
-4124 -1108
5501 -209
3756 3378
-6892 5419
-6342 5283
4144 -772
-2900 3461
1987 6727
-5164 9990
5139 -5243
-9005 -331
4202 1435
8604 1053
-3945 -3371
-6545 -2992
-622 -4248
-6433 5805
-6331 5007
-7380 879
-9523 -5245
-7532 -5811
-59 -3847
-5447 -5991
5567 -3533
-8704 3778
1082 273
5072 1686
-9215 -4819
-9060 -1192
-6312 3440
7678 7507
-3913 -8357
-6395 -9270
4690 -4423
8817 6911
4513 8792
7111 5639
-8282 -2586
-5466 5819
7865 6210
-7955 -7823
7553 4185
-4969 -1029
-6805 -8799
7751 -5664
6143 -8651
-735 3063
-1886 -2387
1103 7711
-6345 6138
5496 -5915
7597 -6888
7340 9063
-8519 5576
6182 5111
3357 5625
-7617 7184
-7673 -4618
-9176 -9664
2494 9960
-244 4702
-8152 -3654
6650 -3420
1630 9673
-1968 1051
4584 3089
-7917 3511
-4805 -4534
-9684 -868
-1680 563
-7102 -9405
7621 -2143
-9665 -2377
-5485 -7359
4914 1004
-7466 -8969
-5553 1520
7130 428
136 -8590
-5024 2096
9619 1320
5762 7880
5278 9260
-7709 -7078
9519 5173
-2860 -2838
2788 7475
7733 136
3002 -9435
7035 1173
-5400 8276
-5438 -1071
2247 4735
-316 -4835
-2973 4
-354 2201
-237 5177
967 -1838
1784 4881
-2793 8825
7383 -2799
872 -35
-7305 966
4303 -8
-6986 -5618
-7109 -8159
-8714 -8271
-8750 1551
8306 -6175
-5952 4097
5698 5712
-8095 1721
5636 118
-7182 3496
5169 -2641
-177 3909
-9083 -5604
6790 -9128
5129 7779
-3810 9685
5417 -3599
7437 1915
-6209 1389
8 -6960
5594 8299
5778 -4367
-8530 -4168
6708 2358
3860 3647
-9959 -7491
4907 925
1731 6084
-7590 433
-6541 -9518
4123 8576
326 -3069
1022 -2813
-1236 -7992
2892 -721
8662 902
5087 4630
1050 865
-6740 3441
7941 -9874
-2262 -4066
5314 -1051
-4471 -809
-555 1295
2684 -9127
2791 472
7325 -6907
1806 -3752
-9566 -3686
223 -6367
-5052 2414
-7213 -9685
-3633 4107
9519 5486
4381 -5363
4887 -57
8529 3660
2442 -8105
2816 3293
8582 -8575
4017 7277
1317 7345
1751 3631
6379 -7609
9255 4077
2443 -3648
563 7293
8228 9643
-9358 -478
-818 -1717
-6817 1258
-2996 1930
893 -5093
-9587 1184
4807 4726
-6729 3428
8845 -1330
5544 2808
-7199 -686
-1420 -1785
8416 -2207
-9197 -2417
-6068 1605
-6042 4440
3376 -8459
3021 9459
4565 738
8406 -5727
5110 -763
-1789 -950
7424 1286
-8669 -4788
-8555 -6888
8966 -4083
4814 7111
-9400 -3016
3324 4134
-7061 2666
-3377 554
-9756 -5885
-8862 -9004
5502 443
1547 2865
2747 6687
-421 -4865
-8831 3935
-3167 -8813
4042 1790
-3475 -5144
-8911 -7728
-6065 -3558
-3841 -954
-9527 1056
8737 2556
6837 1577
-3627 -3127
-7584 -2211
5787 -2010
6320 -3003
9852 5556
-219 -9692
1664 -4468
-1791 9029
228 -8355
-8534 2898
991 3372
-6263 -78
-2966 7291
-8914 -9599
-3514 -9295
4102 5486
-6704 3716
-7886 8796
-9849 9056
-6606 -9802
-1414 4120
2248 9529
1550 6337
7888 5566
827 -1303
-5909 -5056
-7631 -6499
1620 2437
8397 205
7100 8487
-9036 7764
-816 -3631
4201 5704
-1087 6093
8773 -7338
-4547 -587
2301 -2554
1846 -6621
-9980 7908
-9736 4072
6903 4414
-4349 5840
506 -1122
-194 1072
6612 3042
2655 -1368
9372 -8592
9585 -5861
5592 7810
-9173 -1885
-4017 -4762
6858 -5205
9022 -8171
-8427 -75
-7365 3394
3113 -1040
7110 -4448
63 -6399
-963 -1503
758 7107
1306 -4961
-7823 4295
6303 -4792
-7834 416
-9488 6786
8903 667
-1217 -4978
-4644 -5452
-9744 -5089
6489 -7628
6810 4403
-1715 1185
8278 -3206
730 -4099
-5585 4219
-3390 1928
-6450 -5601
-9023 -6317
-5197 -7290
159 2021
4665 -1279
-7400 3479
-7717 -3246
-9090 7141
2227 1869
4734 8547
-107 5093
3081 8812
-114 4661
1573 6113
2111 8302
-9580 734
-553 -9906
4284 6466
7336 5009
1401 -431
-1723 6514
-8105 -8125
6629 -3145
-4325 -4290
2803 5218
2260 -7035
8341 6701
5105 7016
-6239 4555
-4689 -5047
560 -2350
-453 7170
-3024 -7309
-5552 -4470
5684 3283
-7861 -596
-6272 1036
-8038 -7356
6361 -9229
5414 -9179
6014 136
3683 -1614
3025 3298
111 -4348
-9292 -2692
-2643 9661
377 -5066
5166 1931
3361 2355
-1640 -7548
7230 -1502
2736 2456
-9328 -976
890 -5542
9349 -4639
6998 -1205
-6434 -4547
-476 3066
550 -1828
-8055 1062
-9770 -6336
-9933 -6076
5478 -3518
-4075 -5158
8213 9123
-6985 1335
7093 1618
-7341 5577
9337 -8839
-3287 698
1238 -2367
1352 8887
3450 502
8632 -6061
2740 -6392
-6322 6340
4738 3665
6503 7600
-5460 -7199
-6689 -3501
8284 1257
2005 -8230
5140 -7128
4043 7746
-6554 -9679
7670 9231
-9582 -1633
2003 -2776
2294 7170
1747 7853
-9537 1769
-6045 -5346
-9596 9786
1238 3719
2258 6764
-7184 -5015
-7383 7035
-7835 -9938
4892 -3857
-6701 -5851
-3026 6239
9229 -9742
3770 8330
2677 -2030
5475 -4350
9028 -216
5029 -1001
7868 -7481
1754 -8057
-4947 2706
-8154 3085
-6730 -7725
8545 -9818
6954 -100
-4150 3563
-750 3189
-2695 1469
9535 5419
9403 6234
-8837 -4845
8249 1157
3252 4985
-4268 5198
-4184 5955
-1547 2411
1704 -5079
-9049 -1305
-2363 3145
-9447 6658
4075 -1494
-9341 -1286
8733 4721
7117 6877
-5711 -4116
3422 9278
3865 -4102
-487 -8718
3729 7188
-6813 -2565
3048 -6208
-7240 -4011
3638 -4456
-5014 4684
-3313 -2162
4228 3416
-3026 -1048
832 5038
-5209 9551
-4104 -9466
-9692 -4524
3563 5874
3008 -3922
-707 -8664
-8437 8995
8404 3175
-1566 -7246
-6288 -1191
-8349 -6995
7352 -237
1562 853
-7784 340
5358 -4596
8732 6496
-9241 7534
-5286 8947
-9086 8920
-1730 4423
5291 -9847
-7444 7884
-9167 8931
8286 -3896
9759 831
3779 -7844
828 -8620
3046 -4892
-767 -1088
-3535 1344
-3729 168
2927 8478
-4782 -7351
-188 -3221
-7589 9955
4990 6663
-4341 2885
586 59
-5256 -6973
6398 -9894
-6383 -6299
6642 7002
-1040 -1308
7141 -935
-2397 -9979
-6470 6239
-8285 -2663
8289 -511
5753 -8541
-7672 -5302
-5168 -8861
-7875 -5403
8098 5324
4556 4470
-923 -7003
7427 -481
5449 -7147
-7662 -3858
-7107 -3910
-5322 8298
-5518 541
6359 1001
-8476 -4546
9867 4253
7726 9087
-2736 -4965
5429 5869
5140 -9917
-4854 3826
-8919 465
9653 6564
931 -6930
8172 -832
-1754 5974
-1147 -6255
-9731 5984
-2743 410
2661 2124
-2693 9875
-4169 8800
-9004 2923
7874 6477
3805 1583
7619 -212
-8677 1800
5151 8421
-4271 -631
-3952 1537
-1105 6059
-3225 6610
-9653 1347
-5747 5451
6294 6505
-6412 -332
-3940 1704
-1598 -7978
-8406 3428
2240 -6978
-7193 -3561
9350 -2661
3864 -4142
-274 -7446
546 -1720
-2145 9511
6562 9632
5810 3877
-2426 1959
3752 -1382
7883 -6689
1091 2645
1077 -4298
5903 -8400
-3331 -8837
-4592 6727
-4391 1360
-7078 -6475
-2987 -1920
607 -3114
7145 2718
-8860 3003
-8732 -1411
-9573 -9448
-6481 -3828
4046 2157
9760 -3279
-9539 519
-6197 -1742
3027 8518
6856 -3116
-6875 6356
5467 9639
-141 -7045
961 7476
253 -3202
-2615 -8954
-5227 3472
3820 4956
6625 8959
-5362 -1982
4808 -4120
7685 677
-8084 -7610
-8372 -5255
4082 -5631
4408 -2993
3807 -3813
8230 1177
2349 4387
-9981 6372
-2545 -2763
-8041 5361
3132 3322
4027 -8602
8964 -7862
3654 -7196
-5562 5842
9236 -1610
8236 3270
-3534 -214
-2844 -8249
-6526 -7435
1548 -5995
8364 -1036
8193 2513
883 2623
7216 5215
-2359 5065
9798 2853
8395 5728
218 -5172
7317 -8551
9797 1489
5414 4371
-6877 -3157
-5151 7984
3128 5159
-3087 2445
-2743 4250
-4443 -5918
890 7572
1788 2493
-7896 6261
3117 9486
206 9463
-7381 9119
7435 -2045
-5411 7186
-4372 -3327
-7596 -9544
7131 5137
9418 4684
1090 7656
4537 1495
-4628 -5742
2470 4010
-3538 -2338
1117 -7368
-6058 1668
-7231 7935
3742 -4088
-825 -5870
2663 -1571
9629 8600
9708 -8528
-7432 -6809
4238 -746
9480 -7152
-8588 -9058
-6451 -7118
-4246 3257
155 3792
-4096 -9113
596 -5168
2946 -6893
-7313 5598
6681 7799
-6344 -143
4211 6128
648 8978
3819 7707
-6033 2435
-5980 8720
-3886 -9787
-3510 -6794
-10000 -6311
366 8661
615 -4679
8856 -2568
-6048 5330
-7421 3136
5021 -4408
3099 -5006
2904 -5489
1980 -5491
6940 -6158
3688 -2598
-3397 2356
-2686 -312
3388 -7491
-2731 6564
-7508 7117
9594 -1795
-4840 2104
-8695 7324
-2885 1268
-5828 4033
6473 2032
8311 -2491
-8894 -9802
-6897 -3971
3946 9829
6508 8291
-7428 -7373
3384 7502
4884 6301
-6112 7335
-3245 8174
419 -6194
-7341 643
7036 -7018
1988 7752
-6566 3441
3399 -3593
-3742 2366
-3591 -2491
-1633 -8030
-6810 -9982
4143 6702
9129 5159
-3449 4712
-6832 4390
6413 6098
4693 5403
-7414 -4506
7544 6755
-3211 -410
-4604 -5321
-7674 2659
-4521 5352
119 3377
7409 7041
6786 -7001
-8501 -5084
-7951 3352
6555 -3998
-505 8231
7385 4574
-9410 -3904
4761 4668
4219 -1046
-6839 8474
1066 -6246
-1983 2360
3459 8737
-8047 -5411
6486 -9569
-1151 9184
-6789 -991
6140 4208
6075 6387
146 766
-9115 -946
-5792 -3816
-1498 8385
-761 4287
-446 8544
-3677 6659
-3802 -8473
9225 2351
-7889 7023
7135 -7427
6661 4562
-2474 4280
7344 -7899
3676 -5043
9875 -5596
-1530 -2993
3699 -9264
381 3415
-6383 -1738
-3546 -3004
-4837 5561
-7451 -5604
-8060 -1698
9144 -9900
-5637 9614
-3306 -6762
-7544 -7405
-4146 8087
2876 3233
-4987 -1979
-8024 -1470
8298 6968
3913 -5947
9501 2677
5690 6346
-6105 5432
-4861 -9199
-7804 -8413
7217 -7287
-7475 -1669
-9822 -7086
-8715 -41
929 -9503
6454 7894
9431 6903
-9670 7423
-372 7752
-7308 1574
3774 4566
7186 4446
-991 -4416
-7959 9020
-7961 -1936
-5446 2973
40 -9713
8878 -4732
-360 3734
-9970 -5599
5069 -6435
-8990 3976
3547 874
-1702 2381
6803 -4899
7578 477
-5999 9858
-1289 9872
5247 -3425
1024 5450
-6966 2709
8932 -6252
4569 6780
-7966 8593
