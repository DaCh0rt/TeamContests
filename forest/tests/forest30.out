82367.87 466442827.67
